pythran/middlend.py

```diff
@@ -24,7 +25,12 @@ def refine(pm, node, optimizations):
     pm.apply(ExpandImportAll, node)
     pm.apply(NormalizeTuples, node)
     pm.apply(RemoveFStrings, node)
+
+    pm.apply(LogicOperateToBool, node)
     pm.apply(ExpandBuiltins, node)
+    pm.apply(ExpandBooleanOperator, node)
```

Comment:
I think these two (expand and combine) should be in the optimization pipeline, not in the transformation pipeline